packages/renderer/src/lib/kubernetes-port-forward/PodNameColumn.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+import { handleNavigation } from '/@/navigation';
+import { podsInfos } from '/@/stores/pods';
+import { NavigationPage } from '/@api/navigation-page';
+
+import type { PodInfo } from '../../../../main/src/plugin/api/pod-info';
+
+interface Props {
+  object: { name: string; namespace: string };
+}
+
+let { object }: Props = $props();
+
+let pod: PodInfo | undefined = $derived(
+  $podsInfos
+    .filter(podInfo => podInfo.kind === 'kubernetes')
+    .find(podInfo => podInfo.Name === object.name && podInfo.Namespace === object.namespace),
+);
+
+async function openPodDetails(): Promise<void> {
+  if (!pod) return;
+
+  return handleNavigation({
+    page: NavigationPage.POD,
+    parameters: {
+      kind: encodeURI(pod.kind),
+      name: encodeURI(pod.Name),
+      engineId: encodeURIComponent(pod.engineId),
+    },
+  });
+}
+</script>
+
+<button title="Open pod details" class="hover:cursor-pointer flex flex-col max-w-full" onclick={openPodDetails.bind(undefined)}>
```

Comment:
this entire file says "pod details" / links to pod, but we are also doing deployments, services, etc? shouldn't this be more generic / link to deployment or service or pod, etc?

Reply:
Yes, but for simplicity will be handled in the follow-up PR adding deployment&services